generator/src/templates/theiaPlugins.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I think we can also get vscode-git https://github.com/che-incubator/vscode-git/releases through that mechanism.

And remove fetching it during build: https://github.com/eclipse/che-theia/blob/1d2ae629da37a96325bfafc3e1e6f6d01b455bd7/dockerfiles/theia/build.sh#L30-L34
Also, copying: https://github.com/eclipse/che-theia/blob/1d2ae629da37a96325bfafc3e1e6f6d01b455bd7/dockerfiles/theia/Dockerfile#L99

Reply:
Will check this.

Reply:
Replaced with provided VSCode builtin extension. It works.